apps/trustlab/src/components/Hero/Hero.js

```diff
@@ -0,0 +1,150 @@
+          height: "500px",
+          overflow: "hidden",
+        }}
+      >
+        {slides.map((slide, index) => (
+          <Slide
+            key={slide.id}
+            direction="left"
+            in={activeStep === index}
+            mountOnEnter
+            unmountOnExit
+            timeout={{ enter: 800, exit: 800 }}
+            appear={false}
+          >
+            <Box
+              sx={{
+                position: "absolute",
+                top: 0,
+                left: 0,
+                width: "100%",
+                height: "100%",
+                backgroundImage: slide.backgroundImage?.url
+                  ? `url(${slide.backgroundImage?.url})`
+                  : "none",
+                backgroundSize: "cover",
+                backgroundPosition: "center",
+                color: "#fff",
+              }}
+            >
+              <Box
+                sx={{
+                  width: "100%",
+                  height: "100%",
+                  backgroundColor: "rgba(0, 0, 0, 0.8)",
+                  pointerEvents: "none",
+                  zIndex: 1,
+                }}
+              >
+                <Box
+                  sx={(theme) => ({
+                    pt: 10,
+                    px: { xs: 2.5, sm: 0 },
+                    margin: "0 auto",
+                    maxWidth: theme.contentWidths.values,
+                  })}
+                >
+                  <LexicalRichText
+                    elements={slide.title}
+                    TypographyProps={{
+                      variant: "display1",
+                      gutterBottom: true,
+                    }}
+                  />
+                  <LexicalRichText
+                    elements={slide.subtitle}
+                    TypographyProps={{
+                      variant: "h3",
+                      gutterBottom: true,
+                    }}
+                  />
+                  <LexicalRichText
+                    elements={slide.description}
+                    TypographyProps={{
+                      variant: "subtitle1",
+                      sx: { mt: 2 },
+                    }}
+                  />
+
+                  {slide.href && (
+                    <Button
+                      component={slide.href ? Link : undefined}
+                      href={slide.href}
+                      variant="outlined"
+                      sx={{
+                        backgroundColor: "common.white",
+                        color: "#463E3E",
+                        border: "none",
+                        mt: 2,
+                        "&:hover": {
+                          transform: "scale(0.95)",
+                        },
+                      }}
+                    >
+                      {slide.label}
+                    </Button>
+                  )}
+                </Box>
+              </Box>
+            </Box>
+          </Slide>
+        ))}
+        <Section
```

Comment:
A single Section should be enough... but it shouldn't hold this PR back any further.